python/GafferSceneUI/CameraUI.py

```diff
@@ -276,17 +474,25 @@
 
 			"description",
 			"""
-			To use fStop to compute the lens aperture, we need to know focal length in world units.
-			Since we usually store focal length in millimeters to match how we refer to real focal lengths
-			and apertures, we need to know how to scale focal length into world units.
-
-			We default to a value of 0.1, which scales from millimeters to centimeters.  This matches
-			the default world units of Alembic and USD.  If your world units are decimeters or meters,
-			then pick the corresponding option instead.
-
-			If you are controlling the camera using a field of view instead of focal length, then the
-			default aperture is just 1.  You should pick "Custom" and then pick a scale that matches
-			a realistic aperture size ( ie. 0.036 meters )
+			The scale to convert from focal length units to world space
+			units. Combined with f-stop to calculate the lens aperture.
+
+			For example, take a 50mm focal length, and a scene that
+			uses decimeters (dm) for its world space units. The
+			mm-to-dm conversion would be: `50mm × 0.01 = 0.05dm`.
```

Comment:
Sorry had missed this before. I'm not sure this is a helpful example, as the 50mm confuses things. Isn't it just a case of "If you've used mm for f/l and your scene is in decimetres, pick 'mm to decimetres' in the dropdown?" No-one needs to do a calculation unless they're in FOV mode and need accurate bokeh?